client/state/themes/actions/upload-theme.js

```diff
@@ -9,12 +9,28 @@ import { receiveTheme } from 'calypso/state/themes/actions/receive-theme';
 
 import 'calypso/state/themes/init';
 
+const performThemeUpload = ( siteId, file, onProgress ) =>
+	new Promise( ( resolve, rejectPromise ) => {
```

Comment:
Why don't you call the params as `resolve, reject`, as is the custom?

Reply:
This came from the previous version, but renamed it in febde3d3c16edcfc174bc6a1be15b8de150b38fe